multiraft/multiraft.go

```diff
@@ -908,8 +899,14 @@ func (s *state) sendMessage(groupID uint64, msg raftpb.Message) {
 		if log.V(4) {
 			log.Infof("node %v: connecting to new node %v", s.nodeID, nodeID)
 		}
-		if err := s.addNode(nodeID, groupID); err != nil {
-			log.Errorf("node %v: error adding node %v", s.nodeID, nodeID)
+		if err := s.addNode(nodeID); err != nil {
```

Comment:
We should either call addNode(nodeID) or addNode(nodeID, groupID), but not both.